src/component/ProductCard.js

```diff
@@ -0,0 +1,57 @@
+import {Dimensions, Image, StyleSheet, Text, View} from 'react-native';
+import React from 'react';
+import {stylesConstant} from '../styles/abstracts/abstracts';
+
+const ProductCard = () => {
```

Comment:
Please remove static data from the product card (Use Props instead and make the product Card component reusable. @5usan